src/utils.js

```diff
@@ -50,8 +50,13 @@ async function setUpNpmConfig (registry, strictSSL) {
     rollback: false,
     fund: false,
     'strict-ssl': strictSSL,
-    noproxy: 'registry.npmjs.org'
-  });
+    noproxy: 'registry.npmjs.org',
+  };
+  if (process.env.CA_FILE) {
+    config.cafile = process.env.CA_FILE;
```

Comment:
You could technically do a shorthand version of it, no?
`cafile` is by default null (according to https://docs.npmjs.com/cli/v7/using-npm/config). So you could do `cafile: process.env.CA_FILE | null` up above.

Reply:
perfect! let me try this.